src/segment-loader.js

```diff
@@ -669,6 +644,7 @@ export default class SegmentLoader extends videojs.EventTarget {
     let segment = playlist.segments[mediaIndex];
 
     return {
+      requestId: 'segment-loader-' + Math.random(),
```

Comment:
I think it would be better to do `Math.random().toString(36)` like you do with `decryptionId` in `mediaSegmentRequest`.